Docs/Electrum-guide.md

```diff
@@ -1,48 +1,56 @@
 # Electrum guide
 
-## This is a guide for creating a 2 of 3 multisig wallet with FullyNoded 2 and Electrum, building and signing the psbt with Electrum, then exporting the psbt to FullyNoded 2 for signing and broadcasting
+This is a guide for creating a 2-of-3 multisig wallet with FullyNoded 2 and Electrum, building and signing a PSBT with Electrum, then exporting the PSBT to FullyNoded 2 for signing and broadcasting
 
 ### 1. Create a multisig wallet in FN2 by selecting "multisig" -> "create wallet"
 
 <img src="../Images/Electrum/1_fn2_create_wallet.PNG" alt="" width="250"/> <img src="../Images/Electrum/2_fn2_walletcreated.PNG" alt="" width="250"/>
 
-### 2. Save your recovery words, you will need them
+### 2. Save your recovery words
+
+You will need them!
 
 <img src="../Images/Electrum/3_fn2_backupwords.PNG" alt="" width="250"/>
 
-### 3. Open Electrum and create a new mutlisig wallet
+### 3. Open Electrum and create a new multisig wallet
 
 ![](../Images/Electrum/4_create_wallet.png)
 
-### 4. Choose a 2 of 3
+### 4. Choose a 2-of-3
 
 ![](../Images/Electrum/5_mofn.png)
 
 ### 5. Import your own seed (the recovery phrase from FN2)
 
 ![](../Images/Electrum/6_seed.png)
 
-### 6. Its a BIP39 seed so you need to tap "options" and select "BIP39"
+### 6. It's a BIP39 seed, so you need to tap "options" and select "BIP39"
 
 ![](../Images/Electrum/7_bip39seed.png)
 
 ### 7. Add the recovery phrase from FN2
 
 ![](../Images/Electrum/8_addseed.png)
 
-### 8. Confirm derivation - by default FN2 uses the same derivation as Electrum for multisig wallets, just make sure you select `native segwit multisig (p2wsh)`, this is also the default multisig address format used by FN2 (bech32)
+### 8. Confirm derivation
+
+By default FN2 uses the same derivation as Electrum for multisig wallets, just make sure you select `native segwit multisig (p2wsh)`. This is also the default multisig address format used by FN2 (bech32).
 
 ![](../Images/Electrum/9_derivation.png)
 
-### 9. Electrum derives the extended public key for you to export which is derived from the recovery phrase you just added
+### 9. Electrum derives the extended public key for you to export.
+
+(This is in turn derived from the recovery phrase you just added.)
 
 ![](../Images/Electrum/10_exportVpub.png)
 
 ### 10. Enter cosigner #2
 
 ![](../Images/Electrum/11_enter_cosigner2.png)
 
-### 11. Come back to FN2 and export your 1st public key descriptor which holds all the xpubs you need. You can get this by going to the wallets tab, enabling the Electrum wallet, and tapping the "info" button to export the devices seed and public key descriptors.
+### 11. Come back to FN2 and export your first public key descriptor, which holds all the xpubs you need
+
+You can get this by going to the wallets tab, enabling the Electrum wallet, and tapping the "info" button to export the device's seed and public key descriptors.
```

Comment:
The info button no longer exists, no big deal but we should go through and update these guides to match the latest UI.